### PR TITLE
[8.x] Create LogClearCommand.php

### PR DESCRIPTION
**Pull request by redbastie**
This command would be very handy for when the log file gets large after extensive testing.

### CODE REVIEW

File: src/Illuminate/Foundation/Console/LogClearCommand.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $filesystem = new Filesystem;
+        $logFile = storage_path('logs/laravel.log');
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/LogClearCommand.php:32]: The log file might not be there. Also, there may be more than one file.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by taylorotwell** [comment]
I'm not sure Laravel needs a dedicated command for this?

**Comment by GrahamCampbell** [comment]
> log file gets large after extensive testing.

You don't have to log to file. Laravel has other drivers, including one that does not log.

**Comment by redbastie** [comment]
> The log file might not be there.

Which is what the `exists()` check was added for.

> Also, there may be more than one file.

Can be refactored easily to iterate through the entire folder via the `allFiles()` method

> You don't have to log to file.

No you don't, but Laravel does this by default.

> Laravel has other drivers, including one that does not log.

This concern contradicts your statement that there can be many log files.

> I'm not sure Laravel needs a dedicated command for this?

OK. Was worth a try. I'll just make a package for it. 

Thank you for your consideration.

**Comment by redbastie** [comment]
> The log file might not be there.

Which is what the `exists()` check was added for.

> Also, there may be more than one file.

Can be refactored easily to iterate through the entire folder via the `allFiles()` method

> You don't have to log to file.

No you don't, but Laravel does this by default.

> Laravel has other drivers, including one that does not log.

This concern contradicts your statement that there can be many log files.

> I'm not sure Laravel needs a dedicated command for this?

OK. Was worth a try. I'll just make a package for it. 

Thank you for your consideration.

**Pull request closed without merging**